src/components/Logo/__tests__/Logo.test.js

```diff
@@ -22,30 +24,30 @@ describe('components/Logo', () => {
   });
 
   it('should render correctly', () => {
-    const { container } = render(withAppContext(<Logo />));
+    const { container, getByTestId } = render(withAppContext(<Logo tall />));
 
     expect(container.querySelector(`a[href="${configuration.links.home}"]`)).toBeInTheDocument();
     expect(container.querySelector(`img[src="${configuration.logo.url}"]`)).toBeInTheDocument();
-    expect(container.querySelector(`img[height="${configuration.logo.height}"]`)).toBeInTheDocument();
-    expect(container.querySelector(`img[width="${configuration.logo.width}"]`)).toBeInTheDocument();
+    expect(getByTestId('logo')).toHaveStyleRule('height', configuration.logo.height.toString());
```

Comment:
This assertion will check for style rule `height: 80`, instead of `height: 80px`. Is that indended?

Reply:
This is how the test was before, in the end we are only checking if the `configuration` value is set, I agree 80px is nicer, will change.